python/Ganga/GPIDev/Credentials/ICredential.py

```diff
@@ -524,3 +524,4 @@ def timeleftInHMS(self, force_check=False):
         """
         logger.warning("Dummy method used - no information returned")
         return ""
+
```

Comment:
It seems that vim has a different definition of `\n` than most other editors I've used. vim treats it as an end-of-line character rather than a line-separator character. This means that while most other editors will display an empty blank line at the end of the file, vim will not.

The important thing for Git is that there is a `\n` at the end of the file. Vim will inform you of its lack by putting a `[noeol]` in the status bar at the bottom. GitHub will display a missing terminating `\n` with a red `⃠↵`.